cmd/cstor-volume-grpc/api/handler_test.go

```diff
@@ -0,0 +1,95 @@
+package api
+
+import (
+	"testing"
+
+	"github.com/openebs/maya/pkg/client/generated/cstor-volume-grpc/v1alpha1"
+)
+
+type TestUnixSock struct{}
+
+//SendCommand for the dummy unix sock for the test program,
+func (r TestUnixSock) SendCommand(cmd string) ([]string, error) {
+	ret := []string{"OK " + cmd}
+	return ret, nil
+}
+
+func TestRunVolumeSnapCreateCommand(t *testing.T) {
+
+	var sock TestUnixSock
+	APIUnixSockVar = sock
+
+	cases := map[string]struct {
+		expectedError error
+		test          *v1alpha1.VolumeSnapCreateRequest
+	}{
+		"successSnapshotCreate": {
+			expectedError: nil,
+			test: &v1alpha1.VolumeSnapCreateRequest{
+				Version:  ProtocolVersion,
+				Volume:   "dummyvol1",
+				Snapname: "dummysnap1",
+			},
+		},
+	}
+
```

Comment:
Can we add failure or negative test cases, will be more helpful.

Reply:
right now, we do not have any negative test cases to cover. All the errors are from istgt only. we will add it later if we need any.